content/ja/docs/concepts/workloads/pods/sidecar-containers.md

```diff
@@ -0,0 +1,104 @@
+---
+title: サイドカーコンテナ
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.29" state="beta" >}}
+
+サイドカーコンテナは、メインのアプリケーションコンテナと同じ{{< glossary_tooltip text="Pod" term_id="pod" >}}内で実行されるセカンダリーコンテナです。
+これらのコンテナは、主要なアプリケーションコードを直接変更することなく、
+ロギング、モニタリング、セキュリティ、データの同期などの追加サービスや機能を提供することにより、
+アプリケーションコンテナの機能を強化または拡張するために使用されます。
```

Comment:
nit: I think we can put this kind of sentence into one line as hugo will put spaces instead of newlines.